sdk/spring/CHANGELOG.md

```diff
@@ -5,7 +5,23 @@
 - Support Spring Cloud version: 2020.0.3+ and 2021.0.0.
 
 ### Dependency Updates
-Upgrade dependency according to spring-boot-dependencies:2.6.3 and spring-cloud-dependencies:2021.0.0.
+- Upgrade dependency according to spring-boot-dependencies:2.6.3 and spring-cloud-dependencies:2021.0.0.
+
+### Spring Cloud Azure Starter Active Directory
+This section includes changes in `spring-cloud-azure-starter-active-directory` module.
+
+#### Breaking Changes
+- Remove the AAD conditional access filter. [#27727](https://github.com/Azure/azure-sdk-for-java/pull/27727)
```

Comment:
Same here. Add more description about workaround. It's OK to add in PR description.

Reply:
OK